File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int row, s, col;

    //dont want inpt more than 8

    do
    {
        //take input
        row = get_int("rows: ");
    }
    while (row > 8 || row <= 0);
    //pattern

    for (s = 1; s <= row; s++)

    {
        //print spaces
        for (col = 1; col <= row - s; col++)
        {
            printf(" ");
        }

        //print#
        for (col = 1; col <= s; col++)
        {
            printf("#");
        }
        // bring cursor to the next line
        printf("\n");
    }

}

File: Python/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    for j in range(height + i + 3):
        if j == height or j == height + 1 or i + j < height - 1:
            print(" ", end="")
        else:
            print("#", end="")

    print("")

